eZ/Publish/Core/Persistence/Cache/ContentHandler.php

```diff
@@ -107,9 +107,14 @@ public function loadContentInfoByRemoteId($remoteId)
      */
     public function loadVersionInfo($contentId, $versionNo)
     {
-        $this->logger->logCall(__METHOD__, array('content' => $contentId, 'version' => $versionNo));
+        $cache = $this->cache->getItem('version', 'info', $contentId, $versionNo);
```

Comment:
I think this is part of content domain.

And given the amount of code out there I would rather consider something like
- 'content/versioninfo/$id/$version'
- or even 'content/info/$id/versioninfo/$version' to make sure existing content info clear calls work _(custom as well as the once we have int the system)_

Reply:
Second one makes total sense. Will update it first thing tomorrow! :+1: